configure.ac

```diff
@@ -85,12 +85,24 @@ dnl      age.
 dnl   6. If any interfaces have been removed since the last public release, then set age
 dnl      to 0.
 
-CLEAN_VERSION=`echo $PACKAGE_VERSION | $SED "s/p.*//"`
-VERSION_MAJOR=`echo $PACKAGE_VERSION | $SED "s/\..*//"`
-VERSION_MINOR=`echo $CLEAN_VERSION | $SED "s/.*\.//"`
 GEN_TOOL="autoconf"
 
-SHARED_VERSION_INFO="$VERSION_MAJOR:$VERSION_MINOR:0"
+LT_CURRENT="1"
+LT_REVISION="30"
+LT_AGE="0"
+
+SHARED_VERSION_INFO="$LT_CURRENT:$LT_REVISION:$LT_AGE"
+
+dnl src/version-metadata.rc values.
+dnl WIN_RC_FILEVERSION and WIN_RC_FILEVERSION_STR are calculated from
+dnl LT_XXX variables as (current-age.age.revision).
+dnl WIN_RC_PRODUCTVERSION and WIN_RC_PRODUCTVERSION_STR are calculated from
+dnl PACKAGE_VERSION.
+
+WIN_RC_FILEVERSION="1,0,30"
+WIN_RC_FILEVERSION_STR="1.0.30"
```

Comment:
@SoapGentoo , i guess it could be calculated, could you help?

Reply:
very hard, because you'd have to compute "backwards-incompatible changes". Let me think about this a bit, for the time being, let's follow the autotools mythbusters instructions.